protected/humhub/modules/comment/widgets/CommentLink.php

```diff
@@ -55,10 +55,6 @@ public function run()
             $this->mode = self::MODE_INLINE;
         }
 
-        if (!$module->canComment($this->object)) {
```

Comment:
The `CommentLink` is used in other places and modules too. We need to make sure that this widget is hidden if we are not allowed to comment. Maybe add the check in `comment.php` here.

Reply:
Done!